mne/viz/ica.py

```diff
@@ -130,6 +131,159 @@ def plot_ica_sources(ica, inst, picks=None, exclude=None, start=None,
     return fig
 
 
+def _ica_properties_skeleton():
+    import matplotlib.pyplot as plt
+    fig = plt.figure()
+    fig.set_facecolor([0.95] * 3)
+    ax = list()
+    ax.append(fig.add_axes([0.05, 0.5, 0.3, 0.45]))
+    ax.append(fig.add_axes([0.5, 0.6, 0.45, 0.35]))
+    ax.append(fig.add_axes([0.5, 0.5, 0.45, 0.1]))
+    ax.append(fig.add_axes([0.05, 0.1, 0.35, 0.3]))
+    ax.append(fig.add_axes([0.5, 0.1, 0.45, 0.25]))
+    return fig, ax
+
+
+def plot_ica_properties(ica, inst, picks=None, axes=None):
+    """Display component properties
+    FIX DOCSTRING
+    """
+    import matplotlib as mpl
+    from scipy import ndimage
+    from ..io.base import _BaseRaw
+    from ..epochs import _BaseEpochs
+    from mne.time_frequency.psd import psd_multitaper
+
+    # current component (with future interactivity in mind)
+    comp_idx = 0
+
+    picks = range(ica.n_components_) if picks is None else picks
+    picks = [picks] if isinstance(picks, int) else picks
+    if not isinstance(inst, (_BaseRaw, _BaseEpochs)):
+        raise ValueError('inst should be an instance of Raw or Epochs,'
+                         ' got %s instead.' % type(inst))
+    if isinstance(picks, int):
+        picks = [picks]
+    if axes is None:
+        fig, axes = _ica_properties_skeleton()
+    else:
+        if not isinstance(axes, list):
+            raise ValueError('axes must be a list of matplotlib axes objects'
+                             ', got %s instead.' % type(axes))
+        is_correct_type = np.array([isinstance(x, mpl.axes._axes.Axes)
+                                   for x in axes])
+        if not np.all(is_correct_type):
+            first_bad = np.where(is_correct_type == False)[0][0]
+            raise ValueError('axes must be a list of matplotlib axes objects'
+                             ', while one of the list elements is %s.' %
+                             type(axes[first_bad]))
+        if not len(axes) == 5:
+            raise ValueError('axes must be a list of length 5, while the'
+                             ' length is %d' % len(axes))
+
+    # calculations
+    # ------------
+    if isinstance(inst, _BaseRaw):
+        # calculate segment length that gives us about 250
+        # segments and then break up continuous signal into segments
+        from ..epochs import segment_raw
+        plot_line_at_zero = False
+        seg_len = np.arange(0.5, 10., 0.5)
+        nice_len = inst._data.shape[1] / (250. * inst.info['sfreq'])
+        seg_len = seg_len[np.argmin(np.abs(seg_len - nice_len))]
+        inst = segment_raw(inst, segment_length=seg_len, verbose=False)
```

Comment:
I also like what eeglab does for continuous files in `pop_prop`. It is not as informative as for epoched data but you get some "birds eye view" instead of nothing.
BTW - I do not set segmentation to 1s - the code above looks for a value from range `np.arange(0.5, 10., 0.5)` that is closest to segmet length that divides that data into 250 chunks. 250 is arbitrary though :)